FindYourRecipe/FindYourRecipe.DataAccess/Repositories/IngredientRecipeRepository.cs

```diff
@@ -16,11 +16,11 @@ public async Task<IngredientRecipe> CreateAsync(int ingredientId, int recipeId,
         {
             IngredientRecipe ingredientRecipe = new IngredientRecipe()
             {
-                IngredientId = ingredientId,
-                RecipeId = recipeId,
+                IngredientId=ingredientId,
+                RecipeId=recipeId,
```

Comment:
The empty spaces you has here are actually helping when reading the code so I suggest to not remove them. 😄